client/src/styles/style.css

```diff
@@ -172,4 +172,34 @@ td {
   font-size: 1.5rem;
   margin-left: .5rem;
 }
+
+body {
```

Comment:
@Kina-rain - can you make this a class or something that only affects the login page? By adding this style in this way, it affects the body for every page